owners/OWNERS.example

```diff
@@ -0,0 +1,124 @@
+/**
+ * Copyright 2019 The AMP HTML Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// AMPHTML uses a custom owners bot to enforce ownership rules across its
+// codebase to ensure that all incoming pull requests are reviewed by the right
+// people. This file is an example of the supported format.
+
+// OWNERS files support JSON5 syntax (see json5.org), a more human-readable and
+// writable variant of JSON which supports comments, unquoted obeject keys,
+// single quotes, and trailing commas. In this file, "this directory" refers to
+// the directory containing the OWNERS.yaml file.
```

Comment:
I suggest changing this to a `@fileoverview` section with [block style](https://jsdoc.app/about-block-inline-tags.html#overview) comments.

Reply:
Done